tests/e2e/hci_mesh_rwx_singlevc_topology.go

```diff
@@ -80,17 +88,22 @@ var _ = ginkgo.Describe("[hci-mesh-rwx-topology] Hci-Mesh-Topology-SingleVc", fu
 		labelsMap = make(map[string]string)
 		accessmode = v1.ReadWriteMany
 		bindingModeWffc = storagev1.VolumeBindingWaitForFirstConsumer
-		//bindingModeImm = storagev1.VolumeBindingImmediate
+		bindingModeImm = storagev1.VolumeBindingImmediate
 
 		//read topology map
-		topologyLength, leafNode, _, _, leafNodeTag2 = 5, 4, 0, 1, 2
+		topologyLength, leafNode, _, leafNodeTag1, leafNodeTag2 = 5, 4, 0, 1, 2
```

Comment:
Please add the description about  5, 4, 0, 1, 2

Reply:
done